certified-connectors/Blackbaud SKY Add-ins/apiDefinition.swagger.json

```diff
@@ -84,6 +84,36 @@
           }
         }
       }
+    },
+    "/virtual/httprequest": {
+      "post": {
+        "operationId": "SendHttpRequest",
+        "summary": "Send an HTTP request",
+        "description": "Sends an HTTP request to the specified SKY API endpoint.",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "body",
+            "in": "body",
+            "schema": {
+              "$ref": "#/definitions/SendHttpRequestMessage"
+            },
+            "description": "An object that describes the request to be sent to the SKY API.",
+            "required": true
+          }
+        ],
+        "responses": {
+          "default": {
```

Comment:
@Blackbaud-BenLambert Thank you so much for submitting an update for your connectors. May I ask you please to define a proper response. Default responses are not allowed, please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/coding-standards#responses)